LonaStudio/CanvasList/CanvasListEditor.swift

```diff
@@ -207,6 +222,13 @@ class CanvasListEditor: NSOutlineView, NSOutlineViewDataSource, NSOutlineViewDel
         return cellView
     }
 
+    private func onChangedData(execute: @escaping () -> Void, undo: @escaping () -> Void) {
```

Comment:
can we rename this to `changeData`? I use `on` in front of names to indicate that they are event/change handlers, but this is more of a utility method for calling a handler.